packages/terra-application-navigation/src/tabs/Tab.module.scss

```diff
@@ -0,0 +1,157 @@
+:local {
+  .tab.is-disabled {
+    cursor: default;
+  }
+
+  .tab-rollup {
+    .tab-inner {
+      max-width: 100%;
+    }
+
+    .tab-rollup-label {
+      align-items: center;
+      display: inline-flex;
+      max-width: 100%;
+    }
+
+    .tab-rollup-text {
+      display: block;
+      flex: 0 0 auto;
+      min-width: 0;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+      word-break: normal;
+      word-wrap: normal;
+    }
+
+    .tab-rollup-icon {
+      flex: 0 0 auto;
+      height: 0.7143rem;
+      margin-left: 0.2857rem;
+      width: 0.7143rem;
+    }
+  }
+
+  .tab,
+  .tab-rollup {
+    background-clip: content-box;
+    box-shadow: var(--terra-application-navigation-tab-box-shadow, inset 1px 0 0 0 #3a6194, inset 2px 0 0 0 #5483c1);
+    padding-left: 2px;
+
+    &:last-of-type {
+      box-shadow: var(--terra-application-navigation-tab-last-box-shadow, inset 1px 0 0 0 #3a6194, inset 2px 0 0 0 #5483c1, inset -1px 0 0 0 #5483c1, inset -2px 0 0 0 #3a6194);
+      padding-right: 2px;
+    }
+  }
+
+  .tab,
+  .tab-rollup {
+    background-color: transparent;
+    border: 0;
+    color: var(--terra-application-navigation-tab-color, #fff);
+    cursor: pointer;
+    display: flex;
+    flex: 0 0 auto;
+    font-family: inherit;
+    font-size: 0.9285714rem;
+    margin: 0;
+    outline: none;
+    position: relative;
+
+    .tab-inner {
+      align-items: center;
+      background-position: top left;
+      background-repeat: no-repeat;
+      background-size: 0 5px;
+      box-sizing: border-box;
+      justify-content: space-between;
+      overflow: hidden;
+      padding-bottom: 1.143rem;
+      padding-left: 2.143rem;
+      padding-right: 2.143rem;
+      padding-top: 1.143rem;
+      text-align: center;
+      text-decoration: none;
+      text-transform: none;
+      transition-duration: 0s;
+      transition-property: background-size;
+      transition-timing-function: ease;
+    }
+
+    .tab-label {
+      display: block;
+      max-width: 100%;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+      word-break: normal;
+      word-wrap: normal;
+    }
+
+    &:hover {
+      .tab-inner {
+        background-color: var(--terra-application-navigation-tab-hover-background-color, rgba(42, 75, 119, 0.3));
+      }
+    }
+
+    &:active {
+      .tab-inner {
+        background-color: var(--terra-application-navigation-tab-active-background-color, rgba(42, 75, 119, 0.8));
+      }
+    }
+
+    &[aria-current='true'] {
+      box-shadow: var(--terra-application-navigation-tab-current-box-shadow, inset 1px 0 0 0 #5483c1, inset 2px 0 0 0 #3a6194);
+
+      .tab-inner {
+        background-color: #3a6194;
```

Comment:
We should make background-color, background-image, and background-size themable.

Reply:
Updated.